atom/browser/ui/tray_icon.cc

```diff
@@ -21,6 +21,9 @@ void TrayIcon::SetTitle(const std::string& title) {
 void TrayIcon::SetHighlightMode(TrayIcon::HighlightMode mode) {
 }
 
+void TrayIcon::SetIgnoreDoubleClickEvents(bool ignore) {
```

Comment:
There is no need to have an empty implementation here. Please remove it.

Reply:
In the "atom/browser/ui/tray_icon.h" the method should only be declared on Mac OS:

```cpp
#if defined(OS_MACOSX)
  // Sets the flag which determines whether to ignore double click events. This
  // only works on macOS.
  virtual void SetIgnoreDoubleClickEvents(bool ignore);
#endif
```

Reply:
Updated! I also made these pure virtual methods in order to get rid of the compile time error. This seemed to be consistent with a few other methods in this class that avoided including the empty implementation in `tray_icon.cc`. Only declaring the methods on Mac would still result with the same error if we didn't make these pure virtual methods.